dotcom-rendering/src/web/components/Liveness.importable.tsx

```diff
@@ -45,9 +54,9 @@ function insert(html: string, switches: Switches) {
 	// We're being sent this string by our own backend, not reader input, so we
 	// trust that the tags and attributes it contains are safe and intentional
 	const blogBody = document.querySelector<HTMLElement>('#liveblog-body');
-	const latestBlock = blogBody?.querySelector('#liveblog-body :first-child');
-	if (!latestBlock || !blogBody) return;
-	blogBody.insertBefore(fragment, latestBlock);
+	if (!blogBody || !topOfBlog) return;
+	// nextSibling? See: https://developer.mozilla.org/en-US/docs/Web/API/Node/insertBefore#example_2
+	blogBody.insertBefore(fragment, topOfBlog.nextSibling);
```

Comment:
We now insert new blocks immediately after the top-of-blog marker. This marker lives below the pinned post, preventing ordering issues